release-notes/7.0/preview/api-diff/preview6/Microsoft.AspNetCore.App/7.0-preview6_Microsoft.AspNetCore.Authentication.md

```diff
@@ -0,0 +1,36 @@
+# Microsoft.AspNetCore.Authentication
+
+``` diff
+ namespace Microsoft.AspNetCore.Authentication {
++    public static class AuthenticationConfigurationProviderExtensions {
++        public static IConfiguration GetSchemeConfiguration(this IAuthenticationConfigurationProvider provider, string authenticationScheme);
++    }
+     public class AuthenticationService : IAuthenticationService {
+-        [AsyncStateMachineAttribute(typeof(AuthenticationService.<AuthenticateAsync>d__14))]
+-        public virtual Task<AuthenticateResult> AuthenticateAsync(HttpContext context, string? scheme);
++        [AsyncStateMachineAttribute(typeof(AuthenticationService.<AuthenticateAsync>d__15))]
++        public virtual Task<AuthenticateResult> AuthenticateAsync(HttpContext context, string? scheme);
+-        [AsyncStateMachineAttribute(typeof(AuthenticationService.<ChallengeAsync>d__15))]
+-        public virtual Task ChallengeAsync(HttpContext context, string? scheme, AuthenticationProperties? properties);
++        [AsyncStateMachineAttribute(typeof(AuthenticationService.<ChallengeAsync>d__16))]
++        public virtual Task ChallengeAsync(HttpContext context, string? scheme, AuthenticationProperties? properties);
+-        [AsyncStateMachineAttribute(typeof(AuthenticationService.<ForbidAsync>d__16))]
+-        public virtual Task ForbidAsync(HttpContext context, string? scheme, AuthenticationProperties? properties);
++        [AsyncStateMachineAttribute(typeof(AuthenticationService.<ForbidAsync>d__17))]
++        public virtual Task ForbidAsync(HttpContext context, string? scheme, AuthenticationProperties? properties);
+-        [AsyncStateMachineAttribute(typeof(AuthenticationService.<SignInAsync>d__17))]
+-        public virtual Task SignInAsync(HttpContext context, string? scheme, ClaimsPrincipal principal, AuthenticationProperties? properties);
++        [AsyncStateMachineAttribute(typeof(AuthenticationService.<SignInAsync>d__18))]
++        public virtual Task SignInAsync(HttpContext context, string? scheme, ClaimsPrincipal principal, AuthenticationProperties? properties);
+-        [AsyncStateMachineAttribute(typeof(AuthenticationService.<SignOutAsync>d__18))]
+-        public virtual Task SignOutAsync(HttpContext context, string? scheme, AuthenticationProperties? properties);
++        [AsyncStateMachineAttribute(typeof(AuthenticationService.<SignOutAsync>d__19))]
++        public virtual Task SignOutAsync(HttpContext context, string? scheme, AuthenticationProperties? properties);
+     }
```

Comment:
Removing this whole class since it's all async attribute changes that we don't want to show.
```suggestion
```